MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: docs/topics/mta-install/proc_revoke-api-key-migrator.adoc

:_newdoc-version: 2.18.5
:_template-generated: 2025-07-31
:_mod-docs-content-type: PROCEDURE

[id="revoke-api-key-migrator_{context}"]

> = Revoke API keys as an migrator

Reviewer: Revoke API keys as a migrator

Author: Fixed the article, thanks!

[role="_abstract"]

> As a Migrator, you can revoke an application programming interface (API) key before its lifespan in {ProductFirstRef} web console.

Reviewer 2: ```suggestion
As a migrator, you can revoke an application programming interface (API) key before it expires in the {ProductFirstRef} web console.
```

.Prerequisites

> Your administrator installed {ProductShortName} 8.2.0 in a Red Hat OpenShift cluster.

Reviewer 2: ```suggestion
* You have installed {ProductShortName} 8.2.0 on a Red{nbsp}Hat OpenShift Container Platform cluster.
```

.Procedure

> . Log in to the {ProductShortName} web console.
> . In the Migration mode, click your username at the top right corner.
> . Select *Tokens*.
> . On the *Tokens* page, click the options icon.
> . Click *Revoke*.

Reviewer 2: ```suggestion
. Log in to the {ProductShortName} web console.
. In the *Migration* mode, in the upper-right corner, click your username.
. Click *Tokens*.
. On the *Tokens* page, click the options icon.
. Click *Revoke*.
```
